src/js/media-controller.js

```diff
@@ -666,9 +667,10 @@ class MediaController extends MediaContainer {
   }
 
   get isPip() {
-    return (
-      this.media && this.media == document.pictureInPictureElement
-    );
+    const pipElement =
+      this.getRootNode().pictureInPictureElement ??
```

Comment:
Double check that it doesn't [throw an error](https://github.com/muxinc/media-chrome/pull/154/files#diff-f093c2ba0a65c5a566a4ad86659ffc5bee29bfbbc241f863bed8f634b7a0f9f7L120) to check this on the shadow root in a browser somewhere. The test suite for this change is big... Every browser, with MC both in and out of the shadow dom, both Fullscreen and PiP, entering and exiting. Any of those combinations could need special handling.

Reply:
I checked Chrome and Safari desktop, and iOS. Didn't see an error being thrown.